printer_zpl2/lib/zpl2.py

```diff
@@ -0,0 +1,335 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016 SYLEAM (<http://www.syleam.fr>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+class Zpl2(object):
```

Comment:
Any reason why you didn't create a python lib out of this? IMO this would totally be useful out of Odoo context.